core/src/main/scripts/importer/validateData.py

```diff
@@ -1274,6 +1274,7 @@ def checkMatchedNormSampleBarcode(self, value):
     def checkVerificationStatus(self, value):
         # if value is not blank, then it should be one of these:
         if self.checkNotBlank(value) and value.lower() not in ('verified', 'unknown', 'na'):
+            # Give warning because not used in front end.
```

Comment:
Thanks. I would phrase the comment as: "Giving only warning instead of error because..."